src/Makefile.qt.include

```diff
@@ -110,6 +111,7 @@ BITCOIN_QT_H = \
   qml/bitcoin.h \
   qml/imageprovider.h \
   qml/nodemodel.h \
+  qml/appmode.h \
```

Comment:
please keep the list in alphabetical order, so we would make the following change to the file:

``` diff
diff --git a/src/Makefile.qt.include b/src/Makefile.qt.include
index 096cd116e..06b642d93 100644
--- a/src/Makefile.qt.include
+++ b/src/Makefile.qt.include
@@ -35,8 +35,8 @@ QT_FORMS_UI = \
   qt/forms/transactiondescdialog.ui
 
 QT_MOC_CPP = \
-  qml/moc_nodemodel.cpp \
   qml/moc_appmode.cpp \
+  qml/moc_nodemodel.cpp \
   qt/moc_addressbookpage.cpp \
   qt/moc_addresstablemodel.cpp \
   qt/moc_askpassphrasedialog.cpp \
@@ -108,10 +108,10 @@ QT_QRC_LOCALE_CPP = qt/qrc_bitcoin_locale.cpp
 QT_QRC_LOCALE = qt/bitcoin_locale.qrc
 
 BITCOIN_QT_H = \
+  qml/appmode.h \
   qml/bitcoin.h \
   qml/imageprovider.h \
   qml/nodemodel.h \
-  qml/appmode.h \
   qml/util.h \
   qt/addressbookpage.h \
   qt/addresstablemodel.h \
```